pkg/parse/parse.go

```diff
@@ -723,56 +723,48 @@ func GetBuildOutput(buildOutput string) (define.BuildOutputOption, error) {
 	}
 	isDir := true
 	isStdout := false
-	typeSelected := false
-	pathSelected := false
-	path := ""
-	tokens := strings.Split(buildOutput, ",")
-	for _, option := range tokens {
-		arr := strings.SplitN(option, "=", 2)
-		if len(arr) != 2 {
+	typeSelected := ""
+	pathSelected := ""
+	for _, option := range strings.Split(buildOutput, ",") {
+		key, value, found := strings.Cut(option, "=")
+		if !found {
 			return define.BuildOutputOption{}, fmt.Errorf("invalid build output options %q, expected format key=value", buildOutput)
 		}
-		switch arr[0] {
+		switch key {
 		case "type":
-			if typeSelected {
-				return define.BuildOutputOption{}, fmt.Errorf("duplicate %q not supported", arr[0])
+			if typeSelected != "" {
+				return define.BuildOutputOption{}, fmt.Errorf("duplicate %q not supported", key)
 			}
-			typeSelected = true
-			switch arr[1] {
+			typeSelected = value
```

Comment:
Why is typeSelected a string now? I don't see where this value would ever be used other than the == "" check so a bool feels easier to use?

Reply:
It'll be useful when more exporter types are introduced, at some point, by someone.  In the meantime I can change the next line to read the value it switches on from this variable instead of `value`.

Reply:
I mean if you will need it sure but it feels weird to change that as side effect of switching to strings.Cut()